packages/command-utils/package.json

```diff
@@ -3,7 +3,11 @@
   "version": "9.1.0",
   "description": "Node CLI utilities made by Instructure Inc.",
   "author": "Instructure, Inc. Engineering and Product Design",
-  "main": "./lib/index.js",
+  "type": "module",
+  "exports": {
+    ".": "./lib/index.js",
+    "./package.json": "./package.json"
+  },
```

Comment:
This is how a standards compliant package using ESM should look like. In the future sadly we will have to add the legacy "module" field https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for